tests/k6/tests/enduser/dialogSystemLabelLog.js

```
@@ -0,0 +1,60 @@
+        expectStatusFor(r).to.equal(201);
+        dialogId = r.json();
+        dialog = d;
+    });
+
+    describe('Update label as enduser', () => {
+        let body = {
+            'label': 'Bin'
+        }
+        let response = putEU('dialogs/' + dialogId + '/systemlabels', body);
+        expectStatusFor(response).to.equal(204);
+        response = getEU('dialogs/' + dialogId + '/labellog');
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)